python/GafferTest/StandardSetTest.py

```diff
@@ -51,7 +51,7 @@ def testRunTimeTyped( self ) :
 		s = Gaffer.StandardSet()
 		self.assertEqual( s.typeName(), "Gaffer::StandardSet" )
 		self.assertEqual( s.staticTypeName(), "Gaffer::StandardSet" )
-		self.assert_( s.isInstanceOf, IECore.TypeId.RunTimeTyped )
+		self.assertTrue( s.isInstanceOf, IECore.TypeId.RunTimeTyped )
```

Comment:
is this doing what we want? `assertTrue(expr, msg=None)` is it actually doing `assertTrue( bool(s.isInstanceOf), msg=IECore.TypeId.RunTimeTyped )`? Or am I just being half asleep.

Reply:
Yep, that was broken all along - good catch. d464d6b.